### ISSUE TITLE
Channel size for stream2

### ISSUE THREAD

**Issue by busekuz** [tengteng95/Pose-Transfer#32]
Hi, thank you for such an interesting project! 
I was trying to understand structure of PATBlocks but I was a bit confused by conv layers parameters.
I am assuming cated_stream2 is indicates that it is a stream of pose code (not the initial image code). 
`        if cated_stream2:
`
`            conv_block += [nn.Conv2d(dim*2, dim*2, kernel_size=3, padding=p, bias=use_bias),
                       norm_layer(dim*2),
                       nn.ReLU(True)]`

`else:`
`conv_block += [nn.Conv2d(dim, dim, kernel_size=3, padding=p, bias=use_bias),
                           norm_layer(dim),
                           nn.ReLU(True)]`

But still I am not sure why we had to set in_channel parameter as "dim * 2" and not directly "dim". Can you explain why?

Thanks!

**Comment by tengteng95** [comment]
Hi, "**cated_stream2**" is a flag which indicates whether the channel number of the pose pathway is **2*dim**.  For the first PATB, the channel number of the pose pathway is simply **dim**. However, the channel number of pose pathway is **2*dim** for i-th PATB (i=2,3,..,9) since it incorporates the updated image code. Please refer to  **Pose Code Update** in Section 3.1.1 in our paper for details. Hence, we set the **cated_stream2**  as False for the first PATB while True for others. Hope it will help.

**Comment by busekuz** [comment]
Thank you very much, I understood it more clearly now.
